include/bluetooth/audio.h

```diff
@@ -1032,6 +1032,27 @@ struct bt_audio_unicast_server_cb {
 	 */
 	int (*release)(struct bt_audio_stream *stream);
 
+	/** @brief Publish Capability callback
```

Comment:
Why?

The callback is called publish_capability, and is a callback requesting the data to publish, not a callback to what have been pulish_ed_.

Reply:
Because the documentation says the callback is called to "read the Published Audio Capabilities".

I would argue that the publishing happens when the device announces that it has these data available (i.e. when the service is set up), not when someone is reading the data.  So this is a call to read the published data.

But this is no biggie, I can live with it as is.